Formula/pdftk-java.rb

```diff
@@ -0,0 +1,22 @@
+class PdftkJava < Formula
+  desc "Port of pdftk in java"
+  homepage "https://gitlab.com/pdftk-java/pdftk"
+  url "https://gitlab.com/pdftk-java/pdftk/-/jobs/artifacts/v3.0.6/raw/build/libs/pdftk-all.jar?job=gradle"
+  version "3.0.6"
+  sha256 "d8f76ff7023ae3eb9258e6ec215e1ade723f1136f2c92588b6081dfd1cd5852d"
```

Comment:
There's something weird happening with the checksum.

Reply:
Indeed, it feels like they are running a periodic job that recreates the artifacts, and that's why the signature changes. How do you suggest to proceed? Should I better checkout git tag and build from source?

Reply:
Yes, probably then…

Reply:
Fixed